pom.xml

```diff
@@ -2767,7 +2767,7 @@
         <module>cdap-distributions</module>
         <module>cdap-test</module>
         <module>cdap-integration-test</module>
-        <module>cdap-ui</module>
+<!--        <module>cdap-ui</module>-->
```

Comment:
why this change?

Reply:
HI ankit , It was a change related to cdap ui folder which does not have POM file but it was not necessary for this commit , i have removed the change pushed again . Please check and merge this PR .